DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestForControllers/TestGitHubExtensionSecurityWebApiAccountControllerAssignRolesToUser.cs

```diff
@@ -0,0 +1,181 @@
+﻿using GitHubExtension.Security.DAL.Infrastructure;
+using GitHubExtension.Security.DAL.Interfaces;
+using GitHubExtension.Security.StorageModels.Identity;
+using GitHubExtension.Security.Tests.Mocks;
+using GitHubExtension.Security.WebApi.Library.Controllers;
+using GitHubExtension.Security.WebApi.Library.Services;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using NSubstitute;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.ModelBinding;
+using System.Web.Http.Results;
+using Xunit;
+
+namespace GitHubExtension.Security.Tests.TestForControllers
+{
+    public class TestGitHubExtensionSecurityWebApiAccountControllerAssignRolesToUser
+    {
+        [Fact]
+        public void CheckUserNotFound()
+        {
+            int gitHubId = 12;
```

Comment:
Unit tests must conform the AAA pattern and must be documented explicitly in each test. For example, see article http://c2.com/cgi/wiki?ArrangeActAssert.